assets/admin/scss/_mixins.scss

```
@@ -29,6 +29,8 @@ $clr_red_900: #7F0A3D;
 $clr_white: #FFFFFF;
 $clr_green: #4caf50;
 
+$clr_purple_100: #EFE9FE;
```

Stylelint (17.14.0)

[error] Unexpected empty line before $-variable (scss/dollar-variable-empty-line-before)

(scss/dollar-variable-empty-line-before)